package.json

```diff
@@ -16,7 +16,7 @@
     "test": "tap test/*.js"
   },
   "bugs": "http://github.com/creationix/jsonparse/issues",
-  "engines": ["node >= 0.2.0"],
+  "engines": ["node >= 4.8.2"],
```

Comment:
@creationix not for the `let` but for the `String.fromCodePoint` which is since 4.8.2. Probably it could be done without bumping but why not to bump?